src/interfaces/chain.cpp

```diff
@@ -258,6 +259,7 @@ class ChainImpl : public Chain
     void initMessage(const std::string& message) override { ::uiInterface.InitMessage(message); }
     void initWarning(const std::string& message) override { InitWarning(message); }
     void initError(const std::string& message) override { InitError(message); }
+    void loadWallet(std::unique_ptr<Wallet> wallet) override { ::uiInterface.LoadWallet(wallet); }
```

Comment:
See this be in `interfaces/node`?

Reply:
Thanks for the explanation.

> the `initMessage`, `initWarning`, `initError`, and `loadWallet` methods will likely move to a new interface

I though it could be `interfaces::Wallet`, the `CreateWalletFromFile` caller would get the wallet interface asap, then request the loading etc and then signal the loading for everyone else.